libclamunrar/qopen.cpp

```diff
@@ -63,7 +63,7 @@ void QuickOpen::Load(uint64 BlockPos)
 {
   if (!Loaded)
   {
-    // If loading for the first time, perform additional intialization.
+    // If loading for the first time, perform additional initialization.
```

Comment:
This is 3rd party code, so as tempting as it is to fix, we should leave it as-is.  If you want, you can submit this upstream to the unrar project https://www.rarlab.com/

Reply:
thx restored original